src/main/java/core/basesyntax/serviceimpl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+
+public class ReaderServiceImpl implements ReaderService {
+    @Override
+    public List<String> readOperations(Path pathToFile) {
+        List<String> result;
+        try (BufferedReader fileReader = new BufferedReader(
+                new FileReader(new File(pathToFile.toUri())))) {
```

Comment:
```suggestion
                new FileReader(new File(pathToFile.toUri())))) {
```
Why do you need .toUri()?